edgelet/build/charts/edge-kubernetes/templates/_helpers.tpl

```diff
@@ -36,6 +36,12 @@ Create chart name and version as used by the chart label.
 provisioning:
   source: "manual"
   device_connection_string: {{ .Values.deviceConnectionString | quote }}
+{{- if .Values.iotedged.certificates }}
+certificates:
+  device_ca_cert: "/etc/edgecerts/device_ca_cert"
+  device_ca_pk: "/etc/edgecerts/device_ca_pk"
+  trusted_ca_certs: "/etc/edgecerts/trusted_ca_certs"
```

Comment:
As discussed offline, it might be worthwhile checking if it is possible to only specify the trusted CA certs and skip the device CA. For public preview though, it may be acceptable to require that all 3 be specified or none.

Reply:
I confirmed that all three fields are needed.